src/selectors/nodes.js

```diff
@@ -255,23 +266,32 @@ export const getNodeDataObject = createSelector(
     }, {})
 );
 
-/**
- * Return the modular pipelines tree with full data for each tree node for display.
- */
 export const getModularPipelinesTree = createSelector(
-  [(state) => state.modularPipeline.tree, getNodeDataObject],
-  (modularPipelinesTree, nodes) => {
+  [
+    (state) => state.modularPipeline.tree,
+    (state) => state.slice.apply,
+    getNodeDataObject,
+    getSlicedPipeline,
+  ],
+  (modularPipelinesTree, isSlicingPipelineApplied, nodes, slicedPipeline) => {
```

Comment:
[Nit] isPipelineSliced

Reply:
hmm i think that name is quite confusing, because with this one we try to highlight whether the slicing Pipeline is applied or not, it's not really about whether the pipeline is sliced or not